.gitmodules

```
@@ -2,3 +2,6 @@
 	path = ghost/core/content/themes/casper
 	url = ../../TryGhost/Casper.git
 	ignore = all
+[submodule "ghost/core/content/themes/source"]
+	path = ghost/core/content/themes/source
+	url = git@github.com:TryGhost/Source.git
```

Can you change this to the following:
```suggestion
	url = ../../TryGhost/Source.git
	ignore = all
```

I don't remember why, but that URL fixes some weird git issues I had before

And `ignore = all` helps stop `source` show up as unstaged in git

---

Updated ✅